### PR TITLE
Caliper Ethereum documentation overhaul

### PR DESCRIPTION
**Pull request by rmarede**
## Checklist
 - [x]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [x]  Documentation

## Issue/User story
Caliper Ethereum documentation was outdated and did not provide information on using the `--caliper-flow-skip-install`  that skips the smart contract installation phase.

## Existing issues
- [x] [GitHub Issues](https://github.com/hyperledger/caliper/issues/1589)

Issue #1589 - Caliper ethereum documentation needs a complete overhaul

## Design of the fix
* Added information on how to use the `--caliper-flow-skip-install` to use pre-deployed contracts. Since the contract configuration on the network config file will differ depending on whether this flag is used, I added a subsection for each use case;
* Added a warning regarding the mandatory usage of `fromAddressSeed` when using more than 1 worker;
* Fixed name mismatches between the network config file keys;
* Added information about the TxStatus type - `getResut()` was not documented;
* Updated references to the old `invokeSmartContract` and `queryState` functions, with the new `sendRequests` function;
* Moved besu-specific warnings to the appropriate sections;
* Removed reference to a "special registry contract", which I haven't found any information about anywhere else.

### CODE REVIEW

**Comment by davidkel** [comment]
@rmarede Many thanks for the contribution, I've asked @aklenik to take a look as he has more knowledge than me in this area but it looks good to me so far.

**Comment by davidkel** [comment]
Going to try closing and re-opening to see if it fixes the DCO issue

**Comment by davidkel** [comment]
@rmarede Due to a guthub problem where DCO wasn't recognised, could you try doing a force push to your PR to see if it now forces github to recognise the DCO please ?

**Comment by rmarede** [comment]
@davidkel you mean like a blank push? like force pushing adding a blank line?

File: docs/vNext/Ethereum_Configuration.md [hunk 1]
```diff
@@ -25,7 +23,7 @@ The page covers the following aspects of using the Ethereum adapter:
 
 # Assembling the Network Configuration File
 
-The JSON network configuration file of the adapter essentially defines which contract are expected to be on the network and which account the adapter should use to deploy the pointed contracts and which account use to invoke them. Contract necessary for the benchmark are automatically deployed at benchmark startup while the special `registry` contract needs to be already deployed on the network. ABI and bytecode of the registry contract are reported in the src/contract/ethereum/registry/registry.json.
+The JSON network configuration file of the adapter essentially defines which contract are expected to be on the network and which account the adapter should use to deploy the pointed contracts and which account use to invoke them.
```
Comment (davidkel) [inline docs/vNext/Ethereum_Configuration.md:26]: `which contract are` should be `which contracts are`

File: docs/vNext/Ethereum_Configuration.md [hunk 2]
```diff
@@ -90,39 +88,41 @@ Unfortunately, HTTP connections are explicitly disallowed, as
...
+## Deployer Address Password
 
 The password to use to unlock [deployer address](#deployer-address). If there isn't an unlock password, this key must be present as empty string. If the deployer address private key is present this is not used.
 
 ```json
 "contractDeployerAddressPassword": "gottacatchemall"
 ```
 
-## Benchmark address
+> Hyperledger Besu does not provide wallet services, so the `contractDeployerPassword` is nor supported and the private key variant must be used.
```
Comment (davidkel) [inline docs/vNext/Ethereum_Configuration.md:115]: `nor` should be `not`

File: docs/vNext/Ethereum_Configuration.md [hunk 3]
```diff
@@ -142,31 +144,56 @@ This configuration takes priority over `fromAddressPassword`.
...
 
-## Contract configuration
-It is the list, provided as a json object, of contracts to deploy on the network before running the benchmark. You should provide a json entry for each contract; the key will represent the contract identifier to invoke methods on that contract.
+## Contract Configuration
+It is the list, provided as a JSON object, of pre-deployed contracts or contracts to deploy on the network before running the benchmark. You should provide a JSON entry for each contract. The key will represent the contract identifier to invoke methods on that contract.
+
+Depending on whether you plan to use pre-deployed contracts or deploy them during the benchmark, the configuration will differ slightly. 
 
-For each key you must provide a JSON object with the `path` field pointing to the [contract definition file](#contract-definition-file).
+> Defining configurations simultaneously for both pre-deployed contracts and contracts to be deployed by Caliper is currently **not supported** and may result in unexpected errors. This is because opting for pre-deployed contracts means the contract installation phase will be skipped.
```
Comment (davidkel) [inline docs/vNext/Ethereum_Configuration.md:170]: For the contract configuration, we still want to refer to the path property. It's required if you want to install the smart contract but not required if the contract is already deployed. This unfortunately was the only reference to path outside of the example

**Review by davidkel — CHANGES_REQUESTED** [review]
Just a couple of minor issues here, looks really good. As this is relevant to the latest version of caliper, could you also take this and apply it to the v0.6.0 directory.
Really a great improvement to the docs, many thanks

File: docs/vNext/Ethereum_Configuration.md (hunk 3 above)
Comment (rmarede) [reply docs/vNext/Ethereum_Configuration.md:170]: there's also a reference to path in line 195, in the appropriate _Contracts to deploy_ section. Isn't that enough?

File: docs/vNext/Ethereum_Configuration.md (hunk 1 above)
Comment (rmarede) [reply docs/vNext/Ethereum_Configuration.md]: done

File: docs/vNext/Ethereum_Configuration.md (hunk 2 above)
Comment (rmarede) [reply docs/vNext/Ethereum_Configuration.md]: done

File: docs/vNext/Ethereum_Configuration.md (hunk 3 above)
Comment (davidkel) [reply docs/vNext/Ethereum_Configuration.md]: yep, must have missed that in the PR, sorry

**Review by davidkel — APPROVED** [review]

**Comment by davidkel** [comment]
@rmarede merged, thank you, really much appreciated

**Pull request merged**